cap/cli.py

```diff
@@ -27,7 +27,28 @@
 from __future__ import absolute_import, print_function
 
 from invenio_base.app import create_cli
-
+from flask import current_app
+from flask.cli import with_appcontext
+from invenio_db import db
+from invenio_files_rest.models import Location
 from .factory import create_app
 
+
 cli = create_cli(create_app=create_app)
+
+
+@cli.group()
```

Comment:
@jirikuncar  I see that in files-REST we have this 
```
@files.command()
@click.argument('name')
@click.argument('uri')
@click.option('--default', is_flag=True, default=False)
@with_appcontext
def location(name, uri, default):
    """Create new location."""
    from .models import Location
    location = Location(name=name, uri=uri, default=default)
    db.session.add(location)
    db.session.commit()
    click.secho(str(location), fg='green')
```
It is almost identical to the one I added. Do we keep them both?